database/migrations/2019_10_06_224327_create_print_jobs_table.php

```diff
@@ -19,7 +19,7 @@ public function up()
             $table->text('filename');
             $table->text('filepath');
             $table->unsignedBigInteger('user_id')->nullable();
-            $table->set('state', array_map(fn ($state) => $state->value, PrintJobStatus::cases()));
+            $table->set('state', ['QUEUED', 'ERROR', 'CANCELLED', 'SUCCESS']);
```

Comment:
Same goes for this set, this could just be an array, which can be cast to the set in the model.

Reply:
Added to #671